src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,25 @@
+@mixin dot-size {
+  height: 20px;
+  width: 20px;
+}
+
+@mixin clock-size {
+  height: 500px;
+  width: 500px;
+}
```

Comment:
It's a same logic, create one mixin and pass variable

Reply:
Also in both cases you setting border-radius for elements, so let's set it here to have reusable cirlce